app/services/AddressLookupConfigBuilderService.scala

```diff
@@ -52,13 +52,20 @@ class AddressLookupConfigBuilderService @Inject()(appConfig: AppConfig) {
     )
 
     val manualAddressEntryConfig = ManualAddressEntryConfig(
-        MandatoryAddressFields(
-          addressLine1 = true,
-          addressLine2 = true,
-          postcode     = true
-        )
+      line1MaxLength = 255,
+      line2MaxLength = 255,
+      line3MaxLength = 255,
+      townMaxLength = 255,
```

Comment:
These are massively larger than what we can use, our limits are 27 characters
Our [regex](https://github.com/hmrc/company-registration/blob/93f8a4aad47762bbfa962dd63115650b049e6ab5/app/models/validation/APIValidation.scala#L153-L156) and [JSON validators](https://github.com/hmrc/company-registration/blob/93f8a4aad47762bbfa962dd63115650b049e6ab5/app/models/validation/APIValidation.scala#L183-L185) in `company-registration` show this